### PR TITLE
Create nfc

### PR DESCRIPTION
**Pull request by tomgxz**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

[nfc-card.up.railway.app](https://nfc-card.up.railway.app/)

### CODE REVIEW

**Comment by tomgxz** [comment]
May or may not have forgotten to put the `.json` file extension on
Fixed now

File: domains/nfc.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Tomgxz",
+        "email": "tomgxz138@gmail.com"
+    },
+    "record": {
+        "CNAME": "is3193tx.up.railway.app"
```
Comment (DEV-DIBSTER) [inline domains/nfc.json:7]: Nothing is being displayed on site.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/nfc.json (hunk 1 above)
Comment (tomgxz) [reply domains/nfc.json:7]: This is due to how railway handles CNAME requests. The current development server is [](https://nfc-card.up.railway.app/tom_gxz)

**Comment by tomgxz** [comment]
The URL doesn't show anything due to the way that Railway handles CNAME requests, in that they need to come from the given URL.

image

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**